eks_infra/create_cluster.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+#/ Usage: 
+#/ export AWS_REGION=<AWS-Region>
+#/ export EC2_KEY_PAIR_NAME=<EC2-Key-Pair-Name>
+#/ ./create_cluster.sh eks_cluster_name eks_version
+
+set -e
+
+# Function to create EC2 key pair
+function create_ec2_key_pair() {
+    aws ec2 create-key-pair \
+    --key-name ${1} \
+    --query 'KeyMaterial' \
+    --output text > ./${1}-KeyPair.pem
+}
```

Comment:
Are we intentionally throwing the .pem file away after using it to create the nodegroup for the first time?

(Makes sense from a security perspective)

Reply:
`create_ec2_key_pair` is meant to be used by cx if keypair name is not supplied. For codebuild infrastructure, we use a pre-defined key pair.